### DISCUSSION TITLE
Option to force square window corners on win 11.

### DISCUSSION THREAD

**Discussion by xlsupport** [f1multiviewer/issue-tracker#582] [Feature requests]
Win11 defaults to rounded corners, but sets square corners when you snap windows. With MV you may snap some windows and use Window/Set position for some others. Then you end up with a messy mix of square & rounded corners. 

To have a uniform view, please add an option to force square corners.
(api: DWMSetWindowAttribute( DWMWA_WINDOW_CORNER_PREFERENCE, DWMWCP_DONOTROUND)

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #44
